aws_doc_sdk_examples_tools/doc_gen.py

```diff
@@ -194,8 +195,9 @@ def for_root(self, root: Path, config: Optional[Path] = None) -> "DocGen":
         return self
 
     @classmethod
-    def from_root(cls, root: Path, config: Optional[Path] = None) -> "DocGen":
-        return DocGen.empty().for_root(root, config)
+    def from_root(cls, root: Path, config: Optional[Path] = None, validation: ValidationConfig = ValidationConfig()
```

Comment:
Try putting these one per line, and see if Black is OK with that?

Reply:
Fixed.